### ISSUE TITLE
making new releases

### ISSUE THREAD

**Issue by ltalirz** [aiidateam/aiida-quantumespresso#395]
@sphuber While preparing a 3.0.0a5 release, I encountered some minor things:

 * when editing only the `setup.json`, the version consistency pre-commit hook rightfully complains, and then changes the version back to be consistent with the one in `__init__.py` (not what I intended, but ok)
 * when editing only the `__init__.py`, the pre-commit hook does *not* complain (bug)
 * the version number also appears in a wide range of test results. do these need to be updated for every release?

```
$ git grep -i "3.0.0a4"
aiida_quantumespresso/__init__.py:__version__ = '3.0.0a4'
setup.json:    "version": "3.0.0a4"
tests/parsers/test_ph/test_ph_default.yml:parser_info: aiida-quantumespresso parser ph.x v3.0.0a4
tests/parsers/test_ph/test_ph_default.yml:parser_version: 3.0.0a4
tests/parsers/test_ph/test_ph_not_converged.yml:parser_info: aiida-quantumespresso parser ph.x v3.0.0a4
tests/parsers/test_ph/test_ph_not_converged.yml:parser_version: 3.0.0a4
tests/parsers/test_ph/test_ph_out_of_walltime.yml:parser_info: aiida-quantumespresso parser ph.x v3.0.0a4
tests/parsers/test_ph/test_ph_out_of_walltime.yml:parser_version: 3.0.0a4
tests/parsers/test_pw/test_pw_default.yml:  parser_info: aiida-quantumespresso parser pw.x v3.0.0a4
tests/parsers/test_pw/test_pw_default.yml:  parser_version: 3.0.0a4
tests/parsers/test_pw/test_pw_default_xml_new.yml:  parser_info: aiida-quantumespresso parser pw.x v3.0.0a4
tests/parsers/test_pw/test_pw_default_xml_new.yml:  parser_version: 3.0.0a4
tests/parsers/test_pw/test_pw_failed_interrupted.yml:parser_info: aiida-quantumespresso parser pw.x v3.0.0a4
tests/parsers/test_pw/test_pw_failed_interrupted.yml:parser_version: 3.0.0a4
tests/parsers/test_pw/test_pw_failed_interrupted_stdout.yml:parser_info: aiida-quantumespresso parser pw.x v3.0.0a4
tests/parsers/test_pw/test_pw_failed_interrupted_stdout.yml:parser_version: 3.0.0a4
tests/parsers/test_pw/test_pw_failed_interrupted_xml.yml:parser_info: aiida-quantumespresso parser pw.x v3.0.0a4
tests/parsers/test_pw/test_pw_failed_interrupted_xml.yml:parser_version: 3.0.0a4
tests/parsers/test_pw/test_pw_failed_out_of_walltime.yml:  parser_info: aiida-quantumespresso parser pw.x v3.0.0a4
tests/parsers/test_pw/test_pw_failed_out_of_walltime.yml:  parser_version: 3.0.0a4
tests/parsers/test_pw/test_pw_failed_scf_not_converged.yml:  parser_info: aiida-quantumespresso parser pw.x v3.0.0a4
tests/parsers/test_pw/test_pw_failed_scf_not_converged.yml:  parser_version: 3.0.0a4
tests/parsers/test_pw/test_pw_relax_success.yml:  parser_info: aiida-quantumespresso parser pw.x v3.0.0a4
tests/parsers/test_pw/test_pw_relax_success.yml:  parser_version: 3.0.0a4
tests/parsers/test_pw/test_pw_vcrelax_fractional_success.yml:  parser_info: aiida-quantumespresso parser pw.x v3.0.0a4
tests/parsers/test_pw/test_pw_vcrelax_fractional_success.yml:  parser_version: 3.0.0a4
tests/parsers/test_pw/test_pw_vcrelax_success.yml:  parser_info: aiida-quantumespresso parser pw.x v3.0.0a4
tests/parsers/test_pw/test_pw_vcrelax_success.yml:  parser_version: 3.0.0a4
tests/parsers/test_pw2wannier90/test_pw2wannier90_default.yml:  parser_info: aiida-quantumespresso parser simple v3.0.0a4
tests/parsers/test_pw2wannier90/test_pw2wannier90_default.yml:  parser_version: 3.0.0a4
```

**Comment by ltalirz** [comment]
Ok, so it turns out that indeed these need to be updated for every release (see #396).

I'd say there are two approaches here:
  1. let the pre-commit hook automatically update the parser version in the test files
  1. make the test not check the parser version

Ideally, one would go with 2. - the tests shouldn't really care about which version of the parser is being used as long as it does what it should.

**Comment by sphuber** [comment]
Thanks Leo. Actually depending on how strict you want to be, the parser test checking that the correct version is being parsed _is_ important, so that's why I am tempted to keep it. Adapting the test data is not a big upheaval, because all you have to do is run `pytest test --force-regen` and they will automatically be updated. Regarding the `setup.json` and `__init__.py` version consistency, that would definitely be a good thing to fix

**Comment by sphuber** [comment]
Fixed by #396
